README.md

```diff
@@ -217,28 +216,14 @@ read the ['Financials' section](#developers) below.*
 
 #### Q: What can people expect to get *paid*?
 
-+ All developers will start on **£120** per day
-  (equates to £15 an hour for 8 hour days.)
++ All developers will start on **£120** per day 
+(equates to £15 an hour for 8 hour days)
 + Commission for Founders & Coders *does not* come out of developers' day rate
 + Within 4 months, developers will undergo a **peer led** day rate review
```

Comment:
Added a shortened version of this and a link to the issue in case people want to know more and _contibute to the conversation_.